packages/graphql-yoga/__tests__/plugin-hooks.spec.ts

```diff
@@ -59,3 +59,49 @@ test('onParams -> setResult to event stream execution result', async () => {
   }
   expect(counter).toBe(2);
 });
+
+test('context value identity stays the same in all hooks', async () => {
```

Comment:
I think we have a similar test in Envelop already, you think we should also test this in Yoga ?

Reply:
yes since onParams does not exist in envelop.